Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py

```diff
@@ -58,3 +61,51 @@ def render_GET(self, request):
             playlists.append({"id": playlist[0], "name": playlist[1], "description": playlist[2], "torrents": torrents})
 
         return json.dumps({"playlists": playlists})
+
+    def render_PUT(self, request):
+        """
+        .. http:put:: /channels/discovered/(string: channelid)/playlists
+
+        Create a new empty playlist with a given name and description. The name and description parameters are
+        mandatory.
+
+            **Example request**:
+
+            .. sourcecode:: none
+
+                curl -X PUT http://localhost:8085/channels/discovered/abcd/playlists
+                --data "name=My fancy playlist&description=This playlist contains some random movies"
+
+            **Example response**:
+
+            .. sourcecode:: javascript
+
+                {
+                    "created": True
+                }
+
+            :statuscode 400: if you are missing the name and/or description parameter
+            :statuscode 404: if the specified channel does not exist
+        """
+        channel_info = self.get_channel_from_db(self.cid)
```

Comment:
I would do the query checks first, then you save a call to the DB if something is off.